nss-day-cohort-45/nutyshell-kicking-the-seat-1
# Cwd articles

# Description
Added the articles modules and got them working.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
1. Run the serve command.
2. Run the json server command.
3. Proceed to the served page.
4. Register/login.
5. Fill out the article fields.
6. Click the Save Article button.
7. Observe the article section.
# Checklist:
